src/test/java/com/inventorsoft/junit/mapper/BookMapperTest.java

```diff
@@ -0,0 +1,163 @@
+package com.inventorsoft.junit.mapper;
+
+import com.inventorsoft.junit.dto.request.CreateBookRequest;
+import com.inventorsoft.junit.dto.request.UpdateBookRequest;
+import com.inventorsoft.junit.dto.response.AuthorResponse;
+import com.inventorsoft.junit.dto.response.BookResponse;
+import com.inventorsoft.junit.model.Author;
+import com.inventorsoft.junit.model.Book;
+import com.inventorsoft.junit.repository.AuthorRepository;
+import com.inventorsoft.junit.repository.BookRepository;
+import com.inventorsoft.junit.service.BookService;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@FieldDefaults(level = AccessLevel.PRIVATE)
+class BookMapperTest {
+
+    static final long AUTHOR_ID = 4L;
+    static final String AUTHOR_NAME = "Eric Jensen";
+    static final LocalDate AUTHOR_BIRTHDAY = LocalDate.of(1980, 1, 28);
+
+    static final long ID = 1L;
+    static final String TITLE = "Brain Based Learning";
+    static final LocalDate RELEASE_DATA = LocalDate.of(2022, 11, 3);
+    static final String DESCRIPTION = "DD";
+
+    @Mock
+    AuthorRepository authorRepository;
+    @Mock
+    AuthorMapper authorMapper;
+    @InjectMocks
+    BookMapper bookMapper;
```

Comment:
I would add empty lines 
```suggestion
    @Mock
    AuthorRepository authorRepository;
    
    @Mock
    AuthorMapper authorMapper;
    
    @InjectMocks
    BookMapper bookMapper;
```